### PR TITLE
Configurable close button

### PR DESCRIPTION
**Pull request by Ceda**

### CODE REVIEW

File: vendor/assets/javascripts/data-confirm-modal.js [hunk 1]
```diff
@@ -151,9 +152,9 @@
     var id = 'confirm-modal-' + String(Math.random()).slice(2, -1);
     var fade = settings.fade ? 'fade' : '';
     var modalClass = options.modalClass ? options.modalClass : settings.modalClass;
+    var modalClose = settings.modalClose;
```
Comment (vjt) [inline vendor/assets/javascripts/data-confirm-modal.js]: Hi, thank you. This should read:

```js
var modalClose = options.modalClose ? options.modalClose : settings.modalClose;
```

So to allow an user to specify a `modalClose` option when instantiating the modal.

**Comment by Ceda** [comment]
@vjt I thought global settings would be enough, but I added the option to set the closeButton from data-attribute

**Comment by vjt** [comment]
Thank you. The only thing that smells here is that the close button, to function, needs to have the 
specific attribute `data-dismiss="modal"`. This is an implementation detail of the modal, that we are leaking out in configuration.

Would your use case be satisfied by having the possibility of customising the _contents_ of the `<button></button>`, and not the whole `<button>` tag?

Thanks!

**Comment by Ceda** [comment]
@vjt Config only button body is better variant ;)

**Comment by vjt** [comment]
Great. Thank you!

**Pull request merged**